template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/App-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Microsoft",
+    "classifications": [ "Common", "Console" ],
+    "name": "Simple Console Application",
+    "generatorVersions": "[1.0.0.0-*)",
+    "description": "A project for creating a command line application that uses the latest C# features and runs on .NET Core on Windows, Linux and macOS.",
+    "groupIdentity": "Microsoft.Common.App",
+    "precedence": "1000",
+    "identity": "Microsoft.Common.App.CSharp.6.0",
+    "shortName": "app",
+    "tags": {
+        "language": "C#",
+        "type": "project"
+    },
+    "sourceName": "Company.Application1",
+    "preferNameDirectory": true,
+    "symbols": {
+        "framework": {
+            "type": "parameter",
+            "description": "The target framework for the project.",
+            "datatype": "choice",
+            "choices": [
+                {
+                    "choice": "net6.0",
+                    "description": "Target net6.0"
+                }
+            ],
+            "replaces": "net6.0",
+            "defaultValue": "net6.0"
+        },
+        "langVersion": {
+            "type": "parameter",
+            "datatype": "text",
+            "description": "Sets the LangVersion property in the created project file",
+            "defaultValue": "",
+            "replaces": "$(ProjectLanguageVersion)"
+        },
+        "no-restore": {
+            "type": "parameter",
+            "datatype": "bool",
+            "description": "If specified, skips the automatic restore of the project on create.",
+            "defaultValue": "false"
+        },
+        "nullable": {
+            "type": "parameter",
+            "datatype": "bool",
+            "defaultValue": "true",
+            "description": "Whether to enable nullable reference types for this project."
+        },
+        "target-framework-override": {
+            "type": "parameter",
+            "description": "Overrides the target framework",
+            "replaces": "target-framework-override",
+            "datatype": "string",
+            "defaultValue": ""
+        },
+        "HostIdentifier": {
+            "type": "bind",
+            "binding": "HostIdentifier"
+        }
+    },
+    "primaryOutputs": [
+        { "path": "Company.Application1.csproj" },
+        {
+            "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+            "path": "Program.cs"
+        }
+    ],
+    "defaultName": "Application1",
+    "postActions": [
+        {
+            "condition": "(!no-restore)",
```

Comment:
Should we add "id"s now or later when we are doing localizations?

Reply:
Created https://github.com/dotnet/templating/issues/3271